satellite/analytics/service.go

```diff
@@ -101,6 +118,54 @@ func (service *Service) enqueueMessage(message segment.Message) {
 	}
 }
 
+func (service *Service) createHubSpotUser(data PayloadUser) {
+
+	payloadBytes, err := json.Marshal(data)
+	if err != nil {
+		service.log.Error("Error message", zap.Error(err))
+	}
+
+	body := bytes.NewReader(payloadBytes)
+
+	req, err := http.NewRequest("POST", "https://api.hubapi.com/crm/v3/objects/contacts?hapikey=1da42564-0598-4456-a1dd-23bccbbb3c39", body)
```

Comment:
this api key should not be in our open source code. It should be added as a config, and we will set the value on the infrastructure side
if this key is being used anywhere, we should delete it and replace it with one that hasn't been publicized